packages/cli/src/commands/integration/types.ts

```diff
@@ -83,3 +83,20 @@ export interface BillingPlan {
   }[];
   disabled?: boolean;
 }
+
+export interface Store {
```

Comment:
Can we call it `Resource` right away?

Reply:
👍 It should be, I missed this while using & updating to the "resource" name everywhere else. Will pick this up tomorrow in the AM.